genesyscloud/architect_schedules/resource_genesyscloud_architect_schedules_test.go

```diff
@@ -105,23 +111,112 @@ func TestAccResourceArchitectSchedules(t *testing.T) {
 	})
 }
 
+func TestAccResourceArchitectSchedulesCreateFailsWhenStartDateIsNotWeekday(t *testing.T) {
```

Comment:
Is it possible for a start date in on a Saturday.  I just want to make sure we don't have problems with an export where  we export something that is on Saturday and then we cant reimport it.  Just want to make sure we are good here.

Reply:
I found schedules with weekend start dates already present in two different orgs. I posted in the Architect Support room for more info on this. I'm assuming they could have been created before this rule was enforced.